alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultCustomCertificateAccessorTest.java

```diff
@@ -0,0 +1,174 @@
+package com.synopsys.integration.alert.database.api;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.alert.common.exception.AlertDatabaseConstraintException;
+import com.synopsys.integration.alert.common.persistence.model.CustomCertificateModel;
+import com.synopsys.integration.alert.database.api.mock.MockCustomCertificateRepository;
+import com.synopsys.integration.alert.database.certificates.CustomCertificateRepository;
+
+public class DefaultCustomCertificateAccessorTest {
+    private final String alias = "alias-test";
+    private final String content = "content-test";
+
+    private CustomCertificateModel expectedCustomCertificateModel = new CustomCertificateModel(alias, content, new Date().toString());
+
+    @Test
+    public void getCertificatesTest() {
+        CustomCertificateRepository customCertificateRepository = new MockCustomCertificateRepository(alias, content, new Date());
```

Comment:
`new Date()` will have a different underlying _time_ every time it is called. We should be careful when comparing these.

Reply:
when I was doing my comparisons I wasn't quite sure how to compare the time stamps so I decided to just compare on the alias and content. Instead, I could create a Date object like I did with alias and content and pass it throughout if you think thats a better alternative